diff --git a/my.clearbanc.com/__tests__/components/FooterOnboarding.spec.js b/my.clearbanc.com/__tests__/components/FooterOnboarding.spec.js
index f1534ebaae..b6a9d53fb8 100644
--- a/my.clearbanc.com/__tests__/components/FooterOnboarding.spec.js
+++ b/my.clearbanc.com/__tests__/components/FooterOnboarding.spec.js
@@ -7,13 +7,6 @@ import analytics from '@/utils/analytics';
 import VueRouter from 'vue-router';
 import { ExperimentNames } from '@/data/experiments';
 
-jest.mock('@vue/composition-api', () => ({
-  __esModule: true,
-  default: 'mockedDefaultExport',
-  onMounted: jest.fn(),
-  computed: jest.fn(),
-}));
-
 const route = {
   path: '/test',
   component: FooterOnboarding,
diff --git a/my.clearbanc.com/__tests__/components/deprecated/connect-saltedge-bank-accounts.spec.js b/my.clearbanc.com/__tests__/components/deprecated/connect-saltedge-bank-accounts.spec.js
index 7ad0e64653..a4907318af 100644
--- a/my.clearbanc.com/__tests__/components/deprecated/connect-saltedge-bank-accounts.spec.js
+++ b/my.clearbanc.com/__tests__/components/deprecated/connect-saltedge-bank-accounts.spec.js
@@ -15,12 +15,6 @@ import {
 } from '../../mocks/saltedge-service-mocks';
 
 Vue.use(Vuex);
-jest.mock('@vue/composition-api', () => ({
-  __esModule: true,
-  default: 'mockedDefaultExport',
-  onMounted: jest.fn(),
-  computed: jest.fn(),
-}));
 window.open = jest.fn();
 const route = {
   path: '/test',
diff --git a/my.clearbanc.com/__tests__/components/forms/FormBasicBusinessDetails.spec.js b/my.clearbanc.com/__tests__/components/forms/FormBasicBusinessDetails.spec.js
index 9948146ca4..715c3561a1 100644
--- a/my.clearbanc.com/__tests__/components/forms/FormBasicBusinessDetails.spec.js
+++ b/my.clearbanc.com/__tests__/components/forms/FormBasicBusinessDetails.spec.js
@@ -10,13 +10,6 @@ import { getAllRevenueBands } from '@/data/signup-revenue-bands';
 import { ExperimentNames } from '@/data/experiments';
 import { localVue } from '../../utils/vue-test-setup';
 
-jest.mock('@vue/composition-api', () => ({
-  __esModule: true,
-  default: 'mockedDefaultExport',
-  onMounted: jest.fn(),
-  computed: jest.fn(),
-}));
-
 describe('Component: FormBasicBusinessDetails', () => {
   let wrapper,
     $store,
diff --git a/my.clearbanc.com/__tests__/components/forms/FormMql.spec.js b/my.clearbanc.com/__tests__/components/forms/FormMql.spec.js
index a2834f9e1d..c52fec0c6b 100644
--- a/my.clearbanc.com/__tests__/components/forms/FormMql.spec.js
+++ b/my.clearbanc.com/__tests__/components/forms/FormMql.spec.js
@@ -9,12 +9,6 @@ import { createBusiness } from '../../utils/dummy-factory';
 
 Vue.use(Vuex);
 
-jest.mock('@vue/composition-api', () => ({
-  __esModule: true,
-  default: 'mockedDefaultExport',
-  onMounted: jest.fn(),
-  computed: jest.fn(),
-}));
 // Spies Ah!
 let dispatchSpy;
 // Data
diff --git a/my.clearbanc.com/__tests__/pages/onboarding-2/offer-selection.spec.js b/my.clearbanc.com/__tests__/pages/onboarding-2/offer-selection.spec.js
index cc04e6d850..41f42360e8 100644
--- a/my.clearbanc.com/__tests__/pages/onboarding-2/offer-selection.spec.js
+++ b/my.clearbanc.com/__tests__/pages/onboarding-2/offer-selection.spec.js
@@ -16,13 +16,6 @@ import {
   createOfferOption,
 } from '../../utils/dummy-factory';
 
-jest.mock('@vue/composition-api', () => ({
-  __esModule: true,
-  default: 'mockedDefaultExport',
-  onMounted: jest.fn(),
-  computed: jest.fn(),
-}));
-
 describe('Page: onboarding-2/offer-selection', () => {
   // Data
   const bizExperiements = {
diff --git a/my.clearbanc.com/__tests__/pages/onboarding-2/select-product.spec.js b/my.clearbanc.com/__tests__/pages/onboarding-2/select-product.spec.js
index 1be9d367a7..d63276a973 100644
--- a/my.clearbanc.com/__tests__/pages/onboarding-2/select-product.spec.js
+++ b/my.clearbanc.com/__tests__/pages/onboarding-2/select-product.spec.js
@@ -9,13 +9,6 @@ import analytics from '@/utils/analytics';
 import VueRouter from 'vue-router';
 import { i18n } from '../../../jest.setup';
 
-jest.mock('@vue/composition-api', () => ({
-  __esModule: true,
-  default: 'mockedDefaultExport',
-  onMounted: jest.fn(),
-  computed: jest.fn(),
-}));
-
 let $route, route2, store, segmentSpy, routerSpy;
 describe('Page: onboarding-2/select-product', () => {
   beforeEach(() => {
